posthog/models/activity_logging/activity_log.py

```diff
@@ -101,6 +104,16 @@ def log_activity(
     activity: str,
     detail: Detail,
 ) -> None:
+    if activity == "updated" and (detail.changes is None or len(detail.changes) == 0):
+        logger.warn(
+            "'updated' activity was logged with no changes and was ignored",
+            team_id=team_id,
+            organization_id=organization_id,
+            user_id=user.id,
+            scope=scope,
+        )
```

Comment:
Logging here seems like it'd generate quite a bit of logs that wouldn't really ever be used, so I'm wondering if we should maybe drop it… CC @guidoiaquinti

Reply:
What's the expected event of volumes? If it's a lot and we don't need high cardinality data we could also use a metric that is cheaper.

In any case: for better parsing `"'updated' activity was logged with no changes and was ignored",` -> `ignore_update_activity_change`

Reply:
When standardising the interface, I think we should move towards "save" buttons that are disabled when there's nothing to save, like in the new insight interface. Hence the traffic should drop to near zero eventually.

Reply:
very unscientifically it is about 1 in 1500

We had about 1400 feature flag changes over a weekend, one of those had an empty array.

It's a weak debug signal anyway, so I don't mind removing it...

Reply:
Let merge this and keep an 👁️ on the volume